src/clean-core/alloc_array.hh

```diff
@@ -25,8 +25,11 @@ struct alloc_array
         _size = size;
         _data = _alloc(size);
 
-        for (size_t i = 0; i < size; ++i)
-            new (placement_new, &this->_data[i]) T();
+        if constexpr (!std::is_trivially_constructible_v<T>)
```

Comment:
shouldn't this be `is_trivially_copyable`?

Reply:
this is checking whether the placement new can be skipped during initial construction, no copies taking place

Reply:
nvm, brainfart